src/main/scala/tech/cryptonomic/conseil/tezos/Model.scala

```diff
@@ -0,0 +1,43 @@
+package tech.cryptonomic.conseil.tezos
+
+import scala.math.BigDecimal
+
+object Model {
```

Comment:
I think this name of object is too generic, basically every case class/object used in the code could belong here.

Reply:
I guess you're right, I couldn't come with anything better... any suggestion?
It's a container for scala classes mapped to conseil table row

`DBTableModel`, `DBTableMapping`?? Not easy to come up with a valid name

Reply:
I like `DBTableMapping`.

Reply:
done